fastlane/lib/fastlane/documentation/markdown_docs_generator.rb

```diff
@@ -82,6 +82,12 @@ def all_actions_from_enhancer
       require 'faraday'
       require 'json'
 
+      if ENV['USE_ENHANCE_TEST_DATA']
```

Comment:
For testing bool environment vars we now have `FastlaneCore::Env.truthy?("var_name")`. It has some additional features, e.g. it allows the env var to be set to `"0"` or `"false"` to disable it.